test/integration/options.js

```diff
@@ -466,6 +466,61 @@ module.exports = function (createFn, setup, dismantle) {
     })
   })
 
+  describe('generateMethods', () => {
+    let app = createFn()
+    let server
+
+    before(done => {
+      setup(err => {
```

Comment:
Codacy has a fix for the issue: [Replace `err` with `(err)`](https://app.codacy.com/gh/florianholzapfel/express-restify-mongoose/pullRequest?prid=8423228)
```suggestion
      setup((err) => {
```